actions/sharedconfig/get.go

```diff
@@ -30,3 +30,26 @@ func (a *Action) get(c *gin.Context) {
 
 	c.JSON(http.StatusOK, makeConfig())
 }
+
+// sharedConfig will return the shared configuration
+// Get shared config godoc
+// @Summary		Get shared config
+// @Description	Get shared config
+// @Tags		Shared-config
+// @Produce		json
+// @Success		200 {object} models.SharedConfig "Shared configuration"
+// @Router		/v1/configs/shared [get]
+// @Security	x-auth-xpub
+func (a *Action) get2(c *gin.Context) {
```

Comment:
IMHO we can do this better, without duplicating ourselves.
When it comes to Swagger instead of whole new method with whole swagger description, all you need to do is to change comment line on the `get(...)` method from:
```go
// @Router		/v1/shared-config [get]
```

to 
```go
// @Router		/v1/configs/shared [get]
// @DeprecatedRouter		/v1/shared-config [get]
```

Reply:
Changed, but separated these into 2 separate methods (one with @Router, the other with @DeprecatedRouter) because of swagger issues